main.js

```diff
@@ -60,17 +60,21 @@ startTimerButton.addEventListener('click', function() {
 logActivityButton.addEventListener('click', logActivityEvents);
 createNewActivityButton.addEventListener('click', returnToActivityForm);
 
+window.onload = function() {
```

Comment:
This looks like the right thing to do, according to Slack reply from Scott. Now we just need to get the rest of it working.